### PR TITLE
Add --no-github-release option for some subcommands

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: #405

### CODE REVIEW

File: release_management/create_release.py [hunk 1]
```diff
@@ -68,12 +68,15 @@ class RustCrates:
     """
 
     @staticmethod
-    def set_up_subcommand(subcmd, subparsers, target_func):
+    def set_up_subcommand(
+        subcmd, subparsers, target_func, add_github_release_option=False
```
Comment (mulkieran) [inline release_management/create_release.py:72]: Make ```add_github_release_option``` a _mandatory_ keyword option, so

```
def set_up_subcommand(
        subcmd, subparsers, target_func, *, add_github_release_option=False
)
```

Note the '*'

**Review by mulkieran — CHANGES_REQUESTED** [review]
Just one small request for what is currently up.

File: release_management/create_release.py (hunk 1 above)
Comment (bgurney-rh) [reply release_management/create_release.py]: Resolved.

**Review by mulkieran — APPROVED** [review]
Code changes look great, but I think you should squash this one into a single commit.

**Comment by bgurney-rh** [comment]
Squashed into a single commit.

**Pull request merged**